ztp/ztp-policy-generator/README.md

```diff
@@ -0,0 +1,318 @@
+# Policy Generator (PolicyGen)
+
+The policy generator (PolicyGen) is a kustomize plugin used to facilitate creating ACM policies from a predefined Custom Resources (CRs). There are 3 main items (Policy Categorization, Source CR policy and PolicyGenTemplate) that PolicyGen rely on to generate the ACM policies and its placement binding and placement rule.
+
+1 - Policy Categorization:
+
+ Any generated policy needs to be placed under one of the following categories; 
+
+   - Common: a policy exist under the common category will be applied to all clusters.
+
+   - Groups: a policy exist under the group category will be applied to group of clusters.
+
+   - Sites: a policy exist under the sites category will be applied to a specific cluster.
+
+2 - Source CR policy:
+
+ The CR that will be used to generate the ACM policy needs to be defined accordingly to the category that will be set to it. 
```

Comment:
Is there actually anything about the source CR that makes it applicable to "common"? This first sentence implies that you have to know which category the CR will go into when writing the source CR but my thinking (and experience with PolicyGen so far) is that PolicyGen takes care of these details. Taking this namespace as an example, if it is listed in the common template then it will be applied to all clusters with the name in the source CR. However if it is included in the group template then it will be applied per group and the name will be automatically updated by PolicyGen to be unique for the group. Both of these transformations can happen from this same "clean" source file. If my understanding is correct we should highlight that here in the readme.

Reply:
well, this is right but also the source CR.spec could be defined as there is no need to update it or have to update otherwise it will be removed. Okay, I will explain more.